examples/rfmip-clear-sky/Makefile

```diff
@@ -53,7 +53,7 @@ rrtmgp_rfmip_lw:   rrtmgp_rfmip_lw.o   $(ADDITIONS) $(RRTMGP_BUILD)/librte.a $(R
 
 rrtmgp_rfmip_lw.o: rrtmgp_rfmip_lw.F90 $(ADDITIONS)
 
-rrtmgp_rfmip_sw:   rrtmgp_rfmip_sw.o   $(ADDITIONS) $(RRTMGP_BUILD)/librte.a $(RRTMGP_BUILD)/librrtmgp.a
+rrtmgp_rfmip_sw:   rrtmgp_rfmip_sw.o   $(ADDITIONS) $(RRTMGP_BUILD)/librrtmgp.a $(RRTMGP_BUILD)/librte.a
```

Comment:
I realize this is harmless but why are the libraries reordered here?

Reply:
This was the recommended fix from the Cray scientist that I worked with.

As far as I can tell, this is a feature/weakness of the Cray linker and other linkers that are partly-based on LLVM and support OpenMP for GPUs. They are just not as robust as their CPU brethren. If the static linkage is not in the right order, the binary cannot be built.